tests/LucidFrame/Console/ConsoleTableTest.php

```diff
@@ -113,4 +116,69 @@ public function testNonBorderedTableWithoutHeader()
 
         $this->assertEquals(trim($nonBorderedTableWithoutHeader), trim($table->getTable()));
     }
+
+    public function testAddHeaders()
+    {
+        $borderedTableDefault = '
++----------+------+
+| Language | Year |
++----------+------+
+| PHP      | 1994 |
++----------+------+';
+
+        $table = new ConsoleTable();
+        $table
+            ->addHeaders(['Language', 'Year'])
+            ->addRow()
+            ->addColumn('PHP')
+            ->addColumn(1994);
+
+        $this->assertEquals(trim($borderedTableDefault), trim($table->getTable()));
+    }
+
+    public function testAddHorizontalLine()
+    {
+        $expectedTable = '
++----------+------+
+| Language | Year |
++----------+------+
+| PHP      | 1994 |
+| C++      | 1983 |
++----------+------+
+|          | 0011 |
++----------+------+';
+
+        $table = new ConsoleTable();
+        $table
+            ->addHeaders(['Language', 'Year'])
+            ->addRow(array('PHP',1994))
+            ->addRow(array('C++', 1983))
+            ->addHorizontalLine()
+            ->addRow(['','0011']);
+
+        $this->assertEquals(trim($expectedTable), trim($table->getTable()));
+    }
+
+
+    public function testAddHorizontalLineWhenBordersAreHidden()
+    {
+        $expectedTable = '
+Language  Year 
+----------------
+ PHP       1994 
+ C++       1983 
+----------------
+           0011';
+
+        $table = new ConsoleTable();
+        $table
+            ->addHeaders(['Language', 'Year'])
+            ->addRow(array('PHP',1994))
+            ->addRow(array('C++', 1983))
+            ->addHorizontalLine()
+            ->addRow(['','0011'])
+            ->hideBorder();
+
+        $this->assertEquals(trim($expectedTable), trim($table->getTable()));
+    }
```

Comment:
Could you please also add this new test cases in example.php so that we can test without phpunit?